lib/Sema/BuilderTransform.cpp

```diff
@@ -459,6 +459,33 @@ class BuilderClosureVisitor
                                  /*isTopLevel=*/true);
   }
 
+  /// If there is a #available in the condition, the 'then' will be
+  /// wrapped in a call to buildLimitedAvailability(_:), if available.
+  /// If this is instead a #unavailability condition, the 'else' will
+  /// be wrapped instead.
+  Expr *wrapInLimitedAvailabilityIfNeeded(IfStmt *ifStmt, Expr *arg, 
+                                          bool isUnavailability = false) {
```

Comment:
Nit: Since only one caller is using the default argument, I think it would be better to make it mandatory.